setup.py

```diff
@@ -68,6 +68,7 @@ def run(self):
     'pandas>=0.24.0,<1.0',
     'psutil>=5.0.0',
     'scikit-learn>=0.20.0',
+    'networkx',
```

Comment:
Do you foresee potential issues with certain networkx versions conflicting with other dependencies?
For now it seems ok since networkx without the [all] option requires hardly any dependencies, but otherwise might consider specifying a minimum networkx version.

Reply:
Good point.

Updated to >=2.3, <3.0